EIPS/eip-TBD.md

```diff
@@ -1,85 +1,57 @@
 ---
-title: NFT royalties with on-chain, decentralized enforcement
-description: TBD
-author: David Huber (@cxkoda) and Arran Schlosberg (@aschlosberg)
+title: On-chain NFT royalty enforcement
+description: Hello world
```

Comment:
should we put a brief description here, or remove?

Reply:
I'll add one before sending off the PR.

Reply:
Added